core/src/apps/ethereum/networks.py.mako

```diff
@@ -67,6 +67,15 @@ def _networks_iterator() -> Iterator[NetworkInfoTuple]:
             "${n.shortcut}",  # symbol
             "${n.name}",  # name
         )
+% endfor
+    elif utils.MODEL_IS_T3T1:
```

Comment:
we don't like elifs in this sort of code, this should be an `if`, the below should be `if model == T2T1`, dtto in other places